relatio/utils.py

```diff
@@ -302,6 +412,44 @@ def prettify(narrative) -> str:
 
     return pretty_narrative
 
+def prettify_narratives(narratives: List[Dict], fix_grammar: bool = False):
+    """ 
+    
+    'Prettifies' a given list of narratives.
+    
+    Converts each narrative from a dictionary to a (grammatically incorrect)
+    human-readable phrase. The grammar can optionally be corrected. It takes
+    roughly one minute to correct the grammars of around 250 narratives.
+
+    Args:
+        narratives (List[Dict]): list of narratives.
+        fix_grammar (bool): if True, the prettified narratives will be 
+            grammatically correct. False by default.
+
+    Returns:
+        pretty_narratives (Counter[str]): a counter over the prettified list of narratives.
+    
+    """
+
+    prettifiable_narratives = []
+    for narrative in narratives:
+        if narrative.get('ARG0') is not None:
+            if narrative.get('B-V') is not None:
+                if narrative.get('ARG1') is not None:
+                    prettifiable_narratives.append(narrative)
+    
+    if fix_grammar:
+        prettifiable_narratives = [handle_selection(n) for n in prettifiable_narratives] # handle the '|'s in narratives
+    
+    pretty_narratives = []
+    for narrative in prettifiable_narratives:    
+        pretty_narratives.append(prettify(narrative))
+
+    if fix_grammar:
+        pretty_narratives = fix_grammars(pretty_narratives)
+    
+    pretty_narratives = Counter(pretty_narratives)
```

Comment:
Returning a `Counter()` is suboptimal in most applications where researchers want to link specific narratives to a document and its metadata. Let's stick to returning lists to preserve the indexing between documents, sentences, and downstream narratives.

(I think the easiest is to modify `prettify()` and remove this wrapper.)